web/src/scalefactors.ts

```diff
@@ -21,7 +21,7 @@ export const ScaleFactors: Array<ScaleFactor> = [
   {
     name: "s00",
     maxRows: 10_000_000,
-    prefix: "v2/100k-2p",
+    prefix: "v2/10k-2p",
```

Comment:
even tiny is 10k-2p now !! Do we want to modify tiny ???

Reply:
what about s00? what I was asking is

s00 and tiny now have the same prefix is that okay? Not sure though how this part works

Reply:
oh, got it. It's ok, we're not currently using it, it's just an informative guideline we mention in the readme file. s00 is the smaller cluster size we have, others are just an addition.